IoTConnect_2.0/IoTConnect_2.0/IoTConnectManager/IoTConnectService.swift

```diff
@@ -374,6 +406,9 @@ extension IoTConnectManager {
                             if dataDevice[keyPath:"d.rc"] as! Int == DeviceSync.Response.OK {
                                 var dictToUpdate = self.dictSyncResponse
                                 if strKey == CommandType.ATTRIBUTE_INFO_UPDATE.rawValue {
+//=======
```

Comment:
remove unwanted comment

Reply:
@sw-ot-deveshmevada Changes are done in new push